android/quest/src/test/java/org/smartregister/fhircore/quest/util/extensions/ConfigExtensionsTest.kt

```diff
@@ -734,4 +734,18 @@ class ConfigExtensionsTest : RobolectricTest() {
     )
     assertNotNull(imageProperties.imageConfig?.decodedBitmap)
   }
+
+  @Test
+  fun testNullCasesUpdatedCorrectlyGivenRowProperty(): Unit = runBlocking {
+    val cardViewProperties = profileConfiguration.views[0] as CardViewProperties
+    val listViewProperties = cardViewProperties.content[0] as ListProperties
+    val columnProperties = listViewProperties.registerCard.views[0] as ColumnProperties
+    loadRemoteImagesBitmaps(
+      listOf(columnProperties.children[0]),
+      computedValuesMap = emptyMap(),
+      registerRepository = registerRepository,
+    )
+
+    assertNotNull("Bitmap should be decoded and set correctly")
```

Comment:
This assertion will always pass. How is it linked to the test?

Reply:
@ellykits  I'll update the test